synapse/rest/client/v1/login.py

```diff
@@ -448,7 +448,10 @@ class CasTicketServlet(RestServlet):
     def __init__(self, hs):
         super(CasTicketServlet, self).__init__()
         self.cas_server_url = hs.config.cas_server_url
-        self.cas_service_url = hs.config.cas_service_url
+        self.cas_service_url = (
```

Comment:
it's confusing that `cas_service_url` means something different here than it does in `CasRedirectServlet`.